gym/spaces/box.py

```diff
@@ -50,11 +50,30 @@ def __init__(self, low, high, shape=None, dtype=np.float32, seed=None):
                 "shape must be provided or inferred from the shapes of low or high"
             )
 
+        # return custom infinite if is integer infinite
+        def _get_inf(dtype):
```

Comment:
As far as I can tell, this function doesn't use any extra information that requires putting it here, right? I'd move it e.g. to the `.../spaces/utils.py` file, nested functions are typically not that great of a thing for readability